hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/HazelcastSqlOperatorTable.java

```diff
@@ -402,6 +408,40 @@ private SqlCall rewriteCall(SqlCall call) {
             return call;
         }
 
+        @SuppressWarnings("CheckStyle")
+        private SqlCall expandView(SqlCall call) {
+            if (call instanceof SqlSelect) {
```

Comment:
Style: If the entire method is in one if-block, it's better to invert the if and return early:
```
if (!(call instanceof SqlSelect)) {
    return call;
}
```